editor/components/post-publish-panel/style.scss

```diff
@@ -44,22 +45,25 @@
 
 .editor-post-publish-panel__link {
 	color: $blue-medium-700;
+	font-weight: 400;
```

Comment:
Any chance we have variables for these, or are they hard-coded everywhere? 🤷‍♂️

Reply:
Nope! Hard-coded everywhere. There aren't that many font-weights technically available, although an audit could be worth doing at some point during the high-shine polish stage.